common/web/input-processor/src/text/prediction/languageProcessor.ts

```diff
@@ -284,7 +286,6 @@ export default class LanguageProcessor extends EventEmitter<LanguageProcessorEve
     let original = this.getPredictionState(-reversion.transformId);
     if(!original) {
       console.warn("Could not apply the Suggestion!");
-      return;
     }
```

Comment:
Either this change will cause crashes on line 294, or `!original` is impossible, in which case this entire block should be deleted.